src/nxdoc/nuxeo-server/core-server/automation/automation-scripting.md

```diff
@@ -568,6 +562,43 @@ print(file);
 
 ## Advanced Use
 
+### Java Objects in Automation Scripting
+
+It is possible to allow specific Java classes to be used via Automation Scripting:
+
+- `java.util.ArrayList`
+- `java.util.Arrays`
+- `java.util.Collections`
+- `java.util.UUID`
+- `org.nuxeo.runtime.transaction.TransactionHelper`
+- `org.nuxeo.ecm.core.api.Blobs`
+- `org.nuxeo.ecm.core.api.impl.blob.StringBlob`
+- `org.nuxeo.ecm.core.api.impl.blob.JSONBlob`
```

Comment:
It could be interesting to not have this list here and leverage explorer for this. As it we don't have to maintain it.